org/qmckl_sherman_morrison_woodbury.org

```diff
@@ -169,15 +170,65 @@ qmckl_exit_code qmckl_sherman_morrison_c(const qmckl_context context,
    
    #+CALL: generate_c_interface(table=qmckl_sherman_morrison_args,rettyp=get_value("FRetType"),fname=get_value("Name"))
 
+   #+RESULTS:
+   #+begin_src f90 :tangle (eval f) :comments org :exports none
+   integer(c_int32_t) function qmckl_sherman_morrison &
+       (context, Dim, N_updates, Updates, Updates_index, breakdown, Slater_inv) &
+       bind(C) result(info)
+
+     use, intrinsic :: iso_c_binding
+     implicit none
+
+     integer (c_int64_t) , intent(in)  , value :: context
+     integer (c_int64_t) , intent(in)          :: Dim
+     integer (c_int64_t) , intent(in)          :: N_updates
+     real    (c_double ) , intent(in)          :: Updates(N_updates*Dim)
+     integer (c_int64_t) , intent(in)          :: Updates_index(N_updates)
+     real    (c_double ) , intent(in)          :: breakdown
+     real    (c_double ) , intent(inout)       :: Slater_inv(Dim*Dim)
+
+     integer(c_int32_t), external :: qmckl_sherman_morrison_c
+     info = qmckl_sherman_morrison_c &
+	    (context, Dim, N_updates, Updates, Updates_index, breakdown, Slater_inv)
+
+   end function qmckl_sherman_morrison
+   #+end_src
+   
```

Comment:
I suppose you autogenerated these !? I hope the autogenerated code is OK now. Can you confirm ?

Reply:
Yes, I autogenerated them. The only things that I needed to change is to remove the 'value' keyword in front of the pointer variables and I had to change the 2 occurrences of 'qmckl_sherman_morrison_f' into 'qmckl_sherman_morrison_c'.

Reply:
OK perfect !
I'll add a new lisp function for this type of script.
Thanks for the feedback.